test/e2e/e2e.go

```diff
@@ -72,25 +72,23 @@ func Initialize() {
 	if ConfigVal.ClusterInstallRequired {
 		err := installCluster()
 		if err != nil {
-			log.Println("error while creating cluster, deleting the cluster", err)
+			// collect cluster diagnostics
+			log.Println("collecting cluster diagnostic details in /tmp folder....")
+			_, err =utils.GetBootstrapClusterDiagnostics()
```

Comment:
There's a lint error in the CI due to this

```suggestion
			_, err = utils.GetBootstrapClusterDiagnostics()
```